src/ServiceBus.AttachmentPlugin.Tests/AzureStorageAttachmentConfigurationTests.cs

```diff
@@ -12,7 +13,15 @@ public void Should_apply_defaults_for_missing_arguments()
             Assert.Equal("connectionString", configuration.ConnectionString);
             Assert.NotEmpty(configuration.ContainerName);
             Assert.NotEmpty(configuration.MessagePropertyToIdentifyAttachmentBlob);
+            Assert.Equal(0, configuration.SasTokensValidInSeconds);
+            Assert.Equal("$attachment.sas.uri", configuration.MessagePropertyForSasUri);
             Assert.True(configuration.MessageMaxSizeReachedCriteria(new Message()));
         }
+
+        [Fact]
+        public void Should_not_accept_invalid_arguments()
```

Comment:
This tests invalid SAS token validity duration, right? Not all the arguments.
Either could be renamed or use InlineData to pass variations to exercise all arguments.
I'm good with both.